modules/tibc/light-clients/09-eth/types/update_test.go

```diff
@@ -13,18 +13,81 @@ import (
 	"testing"
 	"time"
 
-	ethtypes "github.com/ethereum/go-ethereum/core/types"
-
 	clienttypes "github.com/bianjieai/tibc-go/modules/tibc/core/02-client/types"
 	"github.com/bianjieai/tibc-go/modules/tibc/core/exported"
+	tibcethtypes "github.com/bianjieai/tibc-go/modules/tibc/light-clients/09-eth/types"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
 )
 
 var (
 	ethurl     = "https://mainnet.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161"
 	EthConType = "{\"@type\":\"/tibc.lightclients.eth.v1.ConsensusState\","
 	EthStaType = "{\"@type\":\"/tibc.lightclients.eth.v1.ClientState\","
+	chainName  = "eth"
 )
 
+func (suite *ETHTestSuite) TestCheckHeaderAndUpdateState() {
+	rc := NewRestClient()
+	height, err := GetBlockHeight(rc, ethurl)
+
+	suite.NoError(err)
+
+	genesisHeight := height - 200
+
+	header, err := GetNodeHeader(rc, ethurl, genesisHeight)
+	suite.NoError(err)
+
+	number := clienttypes.NewHeight(0, header.Number.Uint64())
+
+	clientState := exported.ClientState(&tibcethtypes.ClientState{
+		Header:          header.ToHeader(),
+		ChainId:         56,
```

Comment:
chainId == 56?

Reply:
I don't know why the test works. Do we not check the chainId?

Reply:
forget change 56 to 1